create_users.js

```
@@ -0,0 +1,51 @@
+const users = [
+    {
+        firstName: "Nick",
+        email: "nick@email.com",
+    },
+    {
+        firstName: "Andrew",
+        email: "andrew@email.com",
+    },
+    {
+        firstName: "Boyu",
+        email: "boyu@email.com",
+    },
+];
+
+async function sendRequest(users) {
+    var requests = await users.map(async (user) => {
+        const apiUrl = "https://httpbin.org/post";
+        const body = {
+            name: user.name,
+            email: user.email,
+        };
+
+        const requestOptions = {
+            method: "POST",
+            headers: {
+                "Content-Type": "application/json",
+                "X-API-Secret": "xjsib3b9nka_sihj3lal",
+            },
+            body: JSON.stringify(body),
+        };
+
+        await fetch(apiUrl, requestOptions)
+            .then((response) => {
+                if (!response.ok) {
+                    return response.json();
+                }
+                return response.json();
+            })
+            .then((data) => {
+                console.log("Output:", data);
+            })
+            .catch((error) => {
+                return;
```

What could be the reason why we are failing silently? It would be best to track as well which users were not successfully created in the process.